packages/fluentui/react-northstar/src/components/Tree/TreeItem.tsx

```diff
@@ -262,6 +263,7 @@ export const TreeItem: ComponentWithAs<'div', TreeItemProps> & FluentComponentSt
                 index,
                 selected,
                 selectable,
+                parent,
```

Comment:
`parent` should be added to `TreeTitleProps` and `TreeTitle.propTypes`

Reply:
Done